inventory.py

```diff
@@ -6,6 +6,9 @@
 
 
 def main():
+    """Main entry point of function.
+    This program is a product inventory management system
+    """
     print('*********************************')
```

Comment:
This printed menu probably should be put into another function in order to be able to reuse it.